Python/sort/Sort_An_Arraysof0s1sand2s.py

```diff
@@ -0,0 +1,39 @@
+#This problems are variation of famous Dutch national flag problem.
+def sort012(arr,n):
+    listofZero = []
+    listofOne = []
+    listofTwo = []
+
+    for i in range(0,n):
+        if arr[i] == 0:
+            listofZero.append(0)
+        elif arr[i] == 1:
+            listofOne.append(1)
+        else:
+            listofTwo.append(2)
+
+    Total = listofZero+listofOne+listofTwo
+    return Total 
+
+
+
+
```

Comment:
Remove blank lines

Reply:
@siddharth25pandey as you suggested I modified the code and readme.md file